### DISCUSSION TITLE
Kein Update nach v23.10.9 möglich - [HTTP ERROR] Bad Request

### DISCUSSION THREAD

**Discussion by fritzCARaldo** [tbnobody/OpenDTU#1578] [Q&A] [answered]
Ich entschuldige mich bereits vorher für meine Unkenntnis.

Bis zu v23.10.9 habe ich sehr regelmäßig die Master-Updates der Firmware vorgenommen. Seither bekomme ich beim Versuch, die neueste "opendtu-generic.bin" zu installieren immer die Fehlermeldung "[HTTP ERROR] Bad Request". 

Ich bin leider absoluter Programmierlaie: Mag / Kann mir jemand helfen? Danke!

**Comment by tbnobody** [comment]
Teile doch mal einen Screenshot von Info --> System --> Speicherinformation

**Reply by fritzCARaldo** [reply]
Soeben erfolgt! Ich hab - wie zugegeben - wirklich keinen Plan davon.

**Comment by fritzCARaldo** [comment]
<img width="1153" alt="Bildschirmfoto 2023-12-18 um 15 52 57" src="https://github.com/tbnobody/OpenDTU/assets/118612225/71da5846-76cb-4791-8f90-c43d16be40aa">
<img width="898" alt="Bildschirmfoto 2023-12-18 um 15 52 29" src="https://github.com/tbnobody/OpenDTU/assets/118612225/103945a3-3c60-4237-bd29-3fec0a0955bb">

**Comment by tbnobody** [comment]
Your problem is, that you have the old partition table. 
As mentioned very big at the start page:
![image](https://github.com/tbnobody/OpenDTU/assets/1753222/fab14f71-3dad-4b70-b352-1178663ab9a1)

https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md

**Reply by fritzCARaldo** [reply]
Ok? Ich dachte, da meine Firmware ja von Oktober ist, wäre ich davon nicht betroffen.

**Reply by tbnobody** [reply, marked as answer]
Deine Oktober Firmware hat zufällig noch in die alte Partitionstabelle gepasst. Aber wie du siehst ist selbst dort der Speicher schon zu 98% belegt....

**Comment by fritzCARaldo** [comment]
Hab es wohl geschafft! Danke!

**Comment by distanzcheck** [comment]
Wie hast Du es genau gemacht ?

**Comment by dirkg173** [comment]
@fritzCARaldo, würde mich auch interessieren, wie Du das gemacht hast.

**Reply by fritzCARaldo** [reply]
Hab mir ein ESP-Programm geladen und die aktuellste Version geflasht.

**Comment by Falkener** [comment]
Falls hier noch jemand mit dem Open Fusion Problem landet:
Es funktioniert wenn man nicht opendtu-generic.bin Datei verwendet, sondern die opendtu-generic_esp32s3_usb.bin
Dann lässt es sich problemlos updaten.

**Reply by Omega13x** [reply]
Das ist ja wohl logisch. Ist ja auch ein S3 im Fusion verbaut. Wenn man da etwas anderes drauf brutzeln will, bricht das Update wegen Inkompatibilität ab. Wenn man Glück hat.

**Reply by Falkener** [reply]
Ja, für den Erfahrenen absolut logisch. Für den Neuling nicht offensichtlich und wo die Erklärung die den Unterschied darstellt liegt auch nicht. Die Leitfäden im Netz erwähnen stets nur opendtu-generic.bin. Leicht falsch zu machen. Ja, zum Glück bricht es ab und macht nichts kaputt!

**Reply by mroenne2022** [reply]
@Falkener : Hier z.B. ist es eigentlich eindeutig gezeigt....
https://github.com/markusdd/OpenDTUFusionDocs/blob/main/pics/ReleaseImages.png
